source/features/quick-comment-hiding.tsx

```diff
@@ -69,10 +69,30 @@ function showSubmenu(event: delegate.Event): void {
 	event.preventDefault();
 }
 
+function addLoadingIcon(event: delegate.Event): void {
+	if (event.delegateTarget.firstElementChild) {
+		return;
+	}
+
+	event.delegateTarget.append(
+		<svg width="16" height="16" viewBox="0 0 16 16" className="anim-rotate ml-2 v-align-text-bottom">
```

Comment:
This icon is not present in Octicons.

Most markup can be `cloneNode` from somewhere, but I'm not sure about that since it will still need modifications.